cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/Arguments.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.etl.api;
+
+import java.util.Map;
+
+/**
+ * Represents arguments passed to the stages in the pipeline.
+ */
+public interface Arguments extends Iterable<Map.Entry<String, String>> {
+
+  /**
+   * Returns true if arguments has specified option, otherwise false is returned.
+   */
+  boolean hasOption(String name);
```

Comment:
I think we can just use 'has' and 'get'. Introducing the term 'Option' may be confusing.